templates/rke2/overlay/opt/corral/rke2/init-server.sh

```diff
@@ -1,15 +1,33 @@
 #!/bin/bash
 
+
+if [ -z "${CORRAL_registry_fqdn}" ]; then
```

Comment:
Same, format

Reply:
This is from Izaac's work. Should probably be addressed his PR, and I can rebase to receive the changes.